bindings/nodejs/tests/client/outputBuilders.spec.ts

```diff
@@ -94,20 +82,12 @@ describe.skip('Output builder methods', () => {
                     amount: '0x32',
                 },
             ],
-            tokenScheme: {
-                type: 0,
-                meltedTokens: '0x0',
-                mintedTokens: '0x32',
-                maximumSupply: '0x64',
-            },
+            // 10 hex encoded
+            tokenScheme: new SimpleTokenScheme('0xa', '0x0', '0xa'),
```

Comment:
What do you think about adding
```JS
export const intToHex = (num: string) =>
    '0x' + BigInt(num).toString(16);
    
export const hexToInt = (num: string) =>
    BigInt(num).toString();
```
?  Does't need to be done in this PR, we can also open an issue (not sure if it works with wasm)

Reply:
Meh im not too worried about this tbh. 
The toString(16) is a nice touch though, didnt know it